hmf-common/src/main/java/com/hartwig/hmftools/common/genome/chromosome/CytoBands.java

```diff
@@ -55,7 +57,7 @@ private void loadRegions(final RefGenomeVersion refGenomeVersion)
         int posEndIndex = 2;
         int nameIndex = 3;
 
-        for(String line : lines)
+        lines.stream().skip(1).forEach(line ->
         {
```

Comment:
Ended up not using this, but when testing it was failing on handling it's data files header-line, hence the tiny drive-by fix. Guess it needs a unit test as well.

Reply:
Added test